### PR TITLE
 #159337174 Refactor Pagination Helper Implementation

### PR DESCRIPTION
**Pull request by SEUNAGBEYE**
#### What does this PR do?
Improve pagination user experience
#### Description of Task to be completed?
- Remove all _by fields and deleted_at.
- Return all records if the limit is greater than the records count.
- Refactor assets get to take the request arguments

#### Screenshots (if appropriate)

screen shot 2018-07-27 at 12 41 17 pm
screen shot 2018-07-27 at 12 42 52 pm
screen shot 2018-07-27 at 12 44 12 pm

#### Postman Collection:
[Activo](https://www.getpostman.com/collections/cf652d4416c0b5510729)

#### Blocker(s):
- [#159203295](https://www.pivotaltracker.com/n/projects/2170023/stories/159203295)

### CODE REVIEW

File: api/utilities/paginator.py [hunk 1]
```diff
@@ -28,6 +30,8 @@ def validate_pagination_args(arg_value, arg_name):
     # Sets limit to a default of 10 if it is not part of the query string
     if arg_name == 'limit' and arg_value == 'None':
         return 10  # Defaults limit to 10 if not provided
+    if arg_name == 'page' and arg_value == 'None':
+        return 1  # Defaults limit to 10 if not provided
```
Comment (frankip) [inline api/utilities/paginator.py]: You have typo in the comments  I think it suppossed to be `Defaults Page to 1 if none is provided`

Comment (SEUNAGBEYE) [reply api/utilities/paginator.py]: @frankip Thank you, that was copied and pasted.

Comment (SEUNAGBEYE) [reply api/utilities/paginator.py]: @frankip Fixed.

File: api/utilities/paginator.py [hunk 2]
```diff
@@ -151,62 +134,9 @@ def pagination_helper(model, schema, extra_query=None):
...
-                    "meta":
-                    pagination_object
-                }
-
-            return function(*args, **kwargs)
-
-        return decorated_funtion
-
-    return decorator
+    return data, pagination_object
```
Comment (Dheavyman) [inline api/utilities/paginator.py]: Please add a new line to the end of file

Comment (SEUNAGBEYE) [reply api/utilities/paginator.py]: @Dheavyman Fixed.

**Comment by SEUNAGBEYE** [comment]
@brayoh This PR is ready to be merged.

**Pull request merged**